### DISCUSSION TITLE
Session timeout issue

### DISCUSSION THREAD

**Discussion by lazka** [danny-avila/LibreChat#2121] [General]
I'm using librechat on multiple devices, and usually use it once per day at on each of them. I still get logged out and have to log in once every week.

Is that on purpose or is that a bug? Either way it seems not ideal to me.

thanks

**Comment by fuegovic** [comment]
It's not a bug, it's a feature! 😉
You can customize that settings with this in the .env:
```sh
REFRESH_TOKEN_EXPIRY=(1000 * 60 * 60 * 24) * 7
```
For more information:
https://docs.librechat.ai/install/configuration/user_auth_system.html#session-expiry-and-refresh-token

**Reply by lazka** [reply]
Then it's a very annoying feature, and not like other software behaves :) (GitHub for example)

I don't want to increase the expiry date as that would decrease the security of the system.

Can't it get a new refresh token once in a while?

**Reply by fuegovic** [reply]
It is in fact pretty standard. Lots of websites work this way, especially when dealing with sensitive information (e.g. API keys)

If you don’t need that extra security you can simply set it to last longer (instead of 7 days you can set it to last a couple of months or a year)

If you don’t want to decrease security too much you could set it to something like 30 days. You will have to log back in when updating anyway.

**Reply by lazka** [reply]
ok, thanks :/

**Reply by valzevul** [reply]
@fuegovic I actually have the same issue on `:latest`. My `.env` has 
```
SESSION_EXPIRY=1000 * 60 * 60 * 24 * 30
REFRESH_TOKEN_EXPIRY=(1000 * 60 * 60 * 24) * 30 * 3
```
but I seem to get logged out every hour or so 🤔

**Reply by valzevul** [reply]
Although seems like my problem might be the one answered here instead:
https://github.com/danny-avila/LibreChat/discussions/572
